### PR TITLE
feat: gives also per target error and not only global eucl when selec…

### PR DESCRIPTION
**Pull request by fantes**
…ting measure:eucll

### CODE REVIEW

File: src/supervisedoutputconnector.h [hunk 1]
```diff
@@ -1053,11 +1053,29 @@ namespace dd
             }
           if (beucll)
             {
-              double meucll = eucll(ad_res, -1);
+              std::tuple<double, std::vector<double>> tmeucll
+                  = eucll(ad_res, -1);
+              double meucll = std::get<0>(tmeucll);
```
Comment (BynaryCobweb) [inline src/supervisedoutputconnector.h:1058]: FYI possible syntax
```cpp
double meucll;
std::vector<double> all_meucll;
std::tie(meucll, all_meucll) = eucll(ad_res, -1);
```
C++ tuple unwrapping :) https://en.cppreference.com/w/cpp/utility/tuple/tie

**Review by BynaryCobweb — APPROVED** [review]

File: src/supervisedoutputconnector.h (hunk 1 above)
Comment (fantes) [reply src/supervisedoutputconnector.h]: ok, it is much nicer :)

Comment (beniz) [reply src/supervisedoutputconnector.h]: image

**Comment by sileht** [comment]
Commit title is a bit long

**Review by sileht — APPROVED** [review]
1000eme pr! 🥇

**Pull request merged**